Makefile

```
@@ -0,0 +1,62 @@
+
+COMPILED_FILES = $(patsubst %,$(BUILD_DIR)/%.scpt,$(MODULE_NAMES))
+TEST_FILES     = $(patsubst %,$(TESTS_DIR)/%.applescript,$(MODULE_NAMES))
+
+# Default target
+all: build
+
+# Build all modules
+build: $(COMPILED_FILES)
+
+# Run all tests
+test: build
```

checkmake (0.2.2)

[warning] Target body for "test" exceeds allowed length of 5 (6).

(maxbodylength)